src/app/services/infokit.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { Http, Headers, Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+import { environment } from '../../environments/environment';
+import { APIService } from './api.service';
+import 'rxjs/Rx';
+
+@Injectable()
+export class InfokitService {
+
+    private _getInfokitActive = environment.baseURL + environment.apiEndpoint + 'infokitactive';
```

Comment:
but `_getInfokitActive` & `_activateInfokit` both start with `environment.baseURL + environment.apiEndpoint`

then, why reconcat the strings when you could reuse it?